docs/contributing/using-instrumenter-api.md

```diff
@@ -116,6 +116,17 @@ The `builder()` method accepts three arguments:
 An `Instrumenter` can be built from several smaller components. The following subsections describe
 all interfaces that can be used to customize an `Instrumenter`.
 
+### Set the instrumentation version and OpenTelemetry schema URL
```

Comment:
CC @theletterf - I've added a new section to the Instrumenter API doc, please take a look at this when you have a moment 🙏